gtsfm/utils/metrics.py

```diff
@@ -422,3 +423,16 @@ def save_metrics_as_json(metrics_groups: Delayed, output_dir: str) -> None:
     """
     for metrics_group in metrics_groups:
         metrics_group.save_to_json(os.path.join(output_dir, metrics_group.name + ".json"))
+
+
+def compute_percentage_change(x: float, y: float) -> float:
+    """Return percentage in representing the regression or improvement of a value x, for new value y.
+
+    Args:
+        x: original value to compare against.
+        y: new value.
+
+    Returns:
+        percentage change.
+    """
+    return (y - x) / (x + EPSILON) * 100
```

Comment:
`abs(y - x) / (abs(x) + EPSILON) * 100`?

Reply:
So I compute here a negative or positive percentage change (and use a red or green color based on the sign). I updated the docstring to clarify that.